server/routes/phase-allocation.js

```diff
@@ -4,10 +4,39 @@ const logger = require('../logger.js');
 const { asyncMiddleware } = require('../error-handler.js');
 const { CreatePhaseSchema } = require('../validation');
 const { expressRequestBodyValidator } = require('../middleware');
-const { createGlobalPhase } = require('../services/phase');
+const { createGlobalPhase, getAllSitePhases } = require('../services/phase');
 
 const router = express.Router();
 
+// Read: Get phases/allocations for site
+router.get(
+  '/:id',
+  [
+    keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health', 'employer'),
+    keycloak.getUserInfoMiddleware(),
+  ],
+  asyncMiddleware(async (req, res) => {
+    const { hcapUserInfo: user } = req;
+    const siteId = parseInt(req.params.id, 10);
+    let result;
+    // TODO: check that user has access to siteId
+    if (siteId) {
```

Comment:
Pretty much just bad requests, which are probably impossible to send through the FE. Leaving it without _any_ error checking on the params feels wrong to me but I've fixed it up so that at least it makes sense (previously it ran a map function on an undefined in the case of !siteId which is no bueno)